### PR TITLE
Added project that use vuetify

### PR DESCRIPTION
**Pull request by rahulkumarsingh73690**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ If you have something that you think belongs here, feel free to reach out to us
 - [Narrandum](https://narrandum.com) - Customer journey mapping tool built using Vue.js, Vuetify, and Feathersjs
 - [diskotek.tv](https://diskotek.tv) - Diskotek - Software and Advertising Agency.
 - [awesometechstack.com](https://awesometechstack.com) - Free website technology stack analyzer, technology trends and rating created with Vue + Vuetify + Nuxt
+- [goonlinetools.com](https://goonlinetools.com/) - Free Online Tools site. Built with Nuxt, Vuetifyjs and Vuejs.
```
Comment (karol-andruszkow) [inline README.md:92]: [BOWWE](https://bowwe.com/) - Business Website Builder

**Review by karol-andruszkow — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rahulkumarsingh73690) [reply README.md:92]: Merge this commit

**Comment by rahulkumarsingh73690** [comment]
Please review this PR @ElijahKotyluk

**Comment by s-rigaud** [comment]
The website is pretty nice but I can't find the source code so I guess the project can be added to the company solution part as it is great and fully functionning by itself.

**Comment by s-rigaud** [comment]
This project is not based on Vuetify anymore (using Tailwind instead)

**Pull request closed without merging**